package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "s47_incorrect_cartoon_chartacter",
+  "version": "1.0.0",
+  "description": "PROJECT DESCRIPTION",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC"
```

Comment:
Add dependencies and devDependencies fields to the package.json file. These fields should list all the packages required to run and develop the application. [important]